# not exactly the struct, but we need some of the useful items:
# https://hanspeterschaub.info/basilisk/Documentation/fswAlgorithms/formationFlying/spacecraftReconfig/spacecraftReconfig.html?highlight=spacecraftreconfig

std_msgs/Header header
float64[] on_time_out_msg